helix-rest/src/test/java/org/apache/helix/rest/server/TestInstancesAccessor.java

```diff
@@ -171,6 +173,11 @@ public void testUpdateInstances() throws IOException {
     clusterConfig = _configAccessor.getClusterConfig(CLUSTER_NAME);
     Assert.assertEquals(clusterConfig.getDisabledInstances().keySet(),
         new HashSet<>(Arrays.asList(CLUSTER_NAME + "localhost_12919")));
+    Assert.assertEquals(clusterConfig.getDisabledInstancesWithInfo().keySet(),
+        new HashSet<>(Arrays.asList(CLUSTER_NAME + "localhost_12919")));
+    Assert.assertEquals(Long.parseLong(
+        clusterConfig.getInstanceHelixDisabledTimeStamp(CLUSTER_NAME + "localhost_12919")),
+        Long.parseLong(clusterConfig.getDisabledInstances().get(CLUSTER_NAME + "localhost_12919")));
```

Comment:
Let's add a test case to cover new controller code parsing the old data format.

Reply:
This test will add oldKey:oldFormat in cluster config znode and controller will read it using getDisabledInstance.